File: android-lib/src/androidTest/java/com/kinvey/androidTest/model/IntegerPrimitiveListInPerson.java
package com.kinvey.androidTest.model;

import com.google.api.client.util.Key;

import java.util.List;

/**
 * Created by yuliya on 09/20/17.
 */

public class IntegerPrimitiveListInPerson extends Person {

    @Key("integerList")
    private List<Integer> integerList;

    public IntegerPrimitiveListInPerson() {
    }

    public IntegerPrimitiveListInPerson(String username) {
        this.username = username;
    }

    public List<Integer> getIntegerList() {
        return integerList;
    }

    public void setIntegerList(List<Integer> integerList) {
        this.integerList = integerList;
    }
}

File: android-lib/src/androidTest/java/com/kinvey/androidTest/model/IntegerPrimitiveListInPerson.kt
package com.kinvey.androidTest.model

import com.google.api.client.util.Key

/**
 * Created by yuliya on 09/20/17.
 */

data class IntegerPrimitiveListInPerson(
    @Key("integerList")
    var integerList: List<Int>? = null
): Person() {
    constructor(username: String?): this() {
        this.username = username
    }
}